app/features/accessibility.js

```diff
@@ -151,23 +152,26 @@ const render = (el, tip = document.createElement('visbug-ally')) => {
 const determineColorContrast = el => {
   // question: how to know if the current node is actually a black background?
   // question: is there an api for composited values?
-  const text      = getStyle(el, 'color')
+  const color = el instanceof SVGElement
```

Comment:
if we're renaming this variable, i'd vote for "foreground" to go with "background" instead of "color." what do you think?

Reply:
I agree with you.  
During researching the contrast for a11y, I found [this site](https://contrastchecker.com) which provides a good reference for this variable name.